src/webgl/p5.Shader.js

```diff
@@ -343,6 +343,9 @@ p5.Shader = class {
       this._renderer.uNMatrix.inverseTranspose(this._renderer.uMVMatrix);
       this.setUniform('uNormalMatrix', this._renderer.uNMatrix.mat3);
     }
+    const cameraMat = this._renderer._curCamera.cameraMatrix;
+    const camMat3x3 = cameraMat.createSubMatrix3x3();
```

Comment:
Typically for matrices used to update normals, we use the [inverse transpose of the transform applied to the vertices,](https://stackoverflow.com/questions/13654401/why-transform-normals-with-the-transpose-of-the-inverse-of-the-modelview-matrix) like we do above for the normal matrix. Does it work if we do the same here? (if both seem to work, it would be interesting logging both values to see if they differ.)

Assuming we do the inverse transpose, we maybe should rename the uniform to something like `uCameraRotation` to make it clear that we're not intending for it to be used to apply the camera to vertex positions; only for rotating normals.